infra/deploy-openfga.sh

```
@@ -0,0 +1,567 @@
+  if [[ -z "$existing_app_json" || "$existing_app_json" == 'null' ]]; then
+    echo "Creating app registration $OPENFGA_APP_DISPLAY_NAME..."
+    existing_app_json="$(az ad app create \
+      --display-name "$OPENFGA_APP_DISPLAY_NAME" \
+      --identifier-uris "$OPENFGA_AUDIENCE" \
+      --sign-in-audience AzureADMyOrg \
+      -o json)"
+    assert_az 'az ad app create'
+  else
+    app_id="$(printf '%s' "$existing_app_json" | python3 -c 'import json,sys; print(json.load(sys.stdin)["appId"])')"
+    echo "App registration already exists ($app_id)"
+    needs_update="$(printf '%s' "$existing_app_json" | python3 - "$OPENFGA_AUDIENCE" <<'PY'
```

Shellcheck (0.11.0)

[error] This redirection overrides piped input. To use both, merge or pass filenames.

(SC2259)